ClothesApp/WebAPI/Controllers/BrandsController.cs

```diff
@@ -0,0 +1,99 @@
+using Application.Dtos;
+using Application.Services;
+using Infrastructure.Exceptions;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebAPI.Controllers;
+
+[ApiController]
+[Route("api/BrandsController")] 
+public class BrandsController : ControllerBase
+{
+    private readonly IBrandService _brandService;   
+    
+    private abstract class Messages
```

Comment:
Why is it abstract and why is it here, why not just move it to a separate file and use it across application